lib/FieldType/DataTransformer/RichTextValueTransformer.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\FieldType\DataTransformer;
+
+use eZ\Publish\API\Repository\FieldType;
+use eZ\Publish\Core\FieldType\RichText\Value;
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * DataTransformer for RichText\Value.
+ */
+class RichTextValueTransformer implements DataTransformerInterface
+{
+    /** @var FieldType */
+    private $fieldType;
+
+    public function __construct(FieldType $fieldType)
+    {
+        $this->fieldType = $fieldType;
+    }
+
+    /**
+     * @param mixed $value
+     *
+     * @return string
+     */
+    public function transform($value)
+    {
+        if (!$value instanceof Value) {
+            return '';
+        }
+
+        return (string) $value;
```

Comment:
If I get that correctly, that means the textarea will be filled with the XML internal format (docbook) which won't be super usable to have a Rich Text editor :)
Instead, the textarea should be filled with `xhtml5edit` representation of the field value

Reply:
Oh, you are right. I wasn't aware we are using two different schemas. The fact we are using some processors doing work "behind the scenes" didn't helped either. Anyway, fixed it.